.github/workflows/action.yaml

```diff
@@ -1,31 +1,33 @@
 on: push
```

Comment:
We should update the actions as Github is complaining about them being out of date.